e2e/run_tests.js

```diff
@@ -3,13 +3,99 @@
 
 /* eslint-disable no-await-in-loop, no-console */
 
+const argv = require('yargs').argv;
 const axios = require('axios');
 const cypress = require('cypress');
 const fse = require('fs-extra');
 const {merge} = require('mochawesome-merge');
 const generator = require('mochawesome-report-generator');
 
-const MAX_FAILED_TITLES = 5;
+runTests();
+
+async function runTests() {
+    // Test directories that requires Cypress chromeWebSecurity disabled
+    const WITH_DISABLED_CHROME_WEB_SECURITY = ['enterprise_saml'];
```

Comment:
I moved `okta_login_spec.js` into the new folder `enterprise_saml` to separate group of test/s that requires different environment setting.